src/lib/ffi/ffi_srp6.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+* (C) 2022 Rostyslav Khudolii
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "botan/bigint.h"
+#include "botan/ffi.h"
+#include "botan/internal/ffi_rng.h"
+#include "botan/internal/ffi_util.h"
+#include "botan/rng.h"
+#include "botan/srp6.h"
+#include "botan/symkey.h"
+
+extern "C" {
+
+using namespace Botan_FFI;
+
+BOTAN_FFI_DECLARE_STRUCT(botan_srp6_server_session_struct,
```

Comment:
Indentation in this file is not matching usual format. Sorry about the weird formatting we use but we should try to be consistent about it. There is an `astyle` config in `src/configs/astyle.rc`

Reply:
Yeah, sorry about that. I used Emacs `c-style-hook` from `src/configs` but apparently it didn't do the job. Fixed.

Reply:
That's weird the config should work (I use it all the time) maybe something is wrong with the checked in version.